sys/nginx/default.conf

```diff
@@ -43,27 +45,41 @@ server {
     add_header X-Content-Type-Options nosniff;
     add_header X-Frame-Options SAMEORIGIN;
     add_header X-XSS-Protection "1; mode=block";
+
+    resolver 127.0.0.11 ipv6=off;
     resolver_timeout 5s;
 
     location / {
         try_files $uri /index.php$is_args$args;
     }
 
+    location ~ ^/stats {
+        default_type 'text/plain';
+        internal;
+
+        content_by_lua_file '/etc/nginx/conf.d/lua/stats.lua';
+    }
+
     location ~ ^/index\.php(/|$) {
-        fastcgi_pass 127.0.0.1:9000;
+        internal;
+
+        set $upstream "127.0.0.1";
+        if ($host = "localhost") {
+            set $upstream "phpfpm";
+        }
+        if ($host = "poser.local") {
```

Comment:
IF inside a location is evil .. consider a map instead

Reply:
done